scripts/build_imagemagick

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set -e
+
+WORK_DIR=/usr/src
+
+function handle_error() {
+  local status=$?
+  if [[ $status > 0 ]]; then
+    echo "🔥 There was a problem building the binary. Operation aborted."
+    exit 1
+  fi
+}
+trap handle_error EXIT SIGINT
+
+function add_xenial_repositories() {
+  cat <<-EOT >> /etc/apt/sources.list
+  deb http://archive.ubuntu.com/ubuntu/ xenial main universe
+  deb http://security.ubuntu.com/ubuntu xenial-security main
+EOT
+}
+
+# add source packages repository to sources.list
+function add_source_repositories() {
+  echo "deb-src http://archive.ubuntu.com/ubuntu/ xenial main universe" >> /etc/apt/sources.list
+  if [[ $STACK == "heroku-20" ]]; then
+    add_xenial_repositories
+  fi
+}
+
+add_source_repositories
```

Comment:
`apt-get build-dep` install packages from source instead of binaries. The Docker images don't provide the repositories to download these dependencies, so we need to add `deb-src` to `sources.list`.

A few packages are not present in the Docker image for Heroku 20. `add_xenial_repositories` appends some repositories to `sources.list` in order to successfully install `libomp5`,  `libomp-10-dev` and `libjasper-dev` which are dependencies for `imagemagick`.